tests/sga.cpp

```diff
@@ -113,13 +113,13 @@ BOOST_AUTO_TEST_CASE(sga_evolve_test)
     // The we test that evolution is deterministic if the
     // seed is controlled.
     std::vector<sga> udas = {
-        sga{10u, .90, 1., 0.1, 1., 2u, "exponential", "gaussian", "tournament", 32u},
-        sga{10u, .90, 1., 0.1, 1., 2u, "sbx", "gaussian", "tournament", 32u},
-        sga{10u, .90, 1., 0.1, 1., 2u, "binomial", "gaussian", "tournament", 32u},
-        sga{10u, .90, 1., 0.1, 1., 2u, "single", "gaussian", "tournament", 32u},
+        //sga{10u, .90, 1., 0.1, 1., 2u, "exponential", "gaussian", "tournament", 32u},
```

Comment:
Are these tests supposed to remain commented out?

Reply:
no, remnants of trials ...